doc/classes/ItemList.xml

```diff
@@ -386,26 +399,35 @@
 	</methods>
 	<members>
 		<member name="allow_reselect" type="bool" setter="set_allow_reselect" getter="get_allow_reselect">
-			If [code]true[/code], the currently selected item may be selected again.
+			If [code]allow_reselect[/code] is set to true, the currently selected item can be selected again.
```

Comment:
The `If [code]true[/code],` construct is the one recommended by the guidelines, so it should be kept for consistency: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-if-true-to-describe-booleans